torch/ao/quantization/quantizer/xpu_inductor_quantizer.py

```diff
@@ -0,0 +1,152 @@
+# mypy: allow-untyped-defs
+import functools
+from typing import Any, Dict, Optional, TYPE_CHECKING
+
+import torch
+from torch.ao.quantization.observer import HistogramObserver, PerChannelMinMaxObserver
+from torch.ao.quantization.quantizer.quantizer import QuantizationSpec
+from torch.ao.quantization.quantizer.x86_inductor_quantizer import (
+    _is_any_annotated,
+    FilterFn,
+    int8_in_int8_out_ops,
+    X86InductorQuantizer,
+)
+from torch.ao.quantization.quantizer.xnnpack_quantizer_utils import QuantizationConfig
+from torch.fx import Node
+
+
+if TYPE_CHECKING:
+    from torch.ao.quantization.qconfig import _ObserverOrFakeQuantizeConstructor
+
+__all__ = [
+    "XPUInductorQuantizer",
+    "get_default_xpu_inductor_quantization_config",
+]
+
+
+@functools.lru_cache
+def get_default_xpu_inductor_quantization_config():
```

Comment:
hi, @jerryzh168  @EikanWang  , I’ve made some updates in the latest commit. Would you mind taking a look？ Your feedback would be greatly appreciated.


Just to provide some context: on the GPU side we prefer using the s8 data dtype over u8. This is because oneDNN has better performance when zero-point is set to 0. So when we use s8+symmetric, we can achieve the optimal performance on GPU. Currently, we've set the qscheme to torch.per_tensor_affine , but users can further adjusts qscheme  as torch.per_tensor_symmetric if they are looking to further improve performance.